eslint.config.js

```diff
@@ -0,0 +1,205 @@
+import path from 'node:path';
+import {fileURLToPath} from 'node:url';
+import {builtinModules} from "node:module";
+
+import {FlatCompat} from "@eslint/eslintrc";
+import tseslint from 'typescript-eslint';
+
+// plugins
+import prettierEslint from 'eslint-plugin-prettier';
+import noOnlyTestsEslint from 'eslint-plugin-no-only-tests';
+import regexpEslint from 'eslint-plugin-regexp'
+const typescriptEslint = tseslint.plugin;
+
+// parsers
+const typescriptParser = tseslint.parser;
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = path.dirname(__filename);
+
+// ref: https://eslint.org/docs/latest/use/configure/migration-guide#using-eslintrc-configs-in-flat-config
+// mimic CommonJS variables -- not needed if using CommonJS
+const compat = new FlatCompat({
+	baseDirectory: __dirname
+});
+
+export default [
+	// If ignores is used without any other keys in the configuration object, then the patterns act as global ignores.
+	// ref: https://eslint.org/docs/latest/use/configure/configuration-files#globally-ignoring-files-with-ignores
+	{
+		ignores: [
+			"**/.*",
+			"**/*.d.ts",
+			"packages/**/*.min.js",
+			"packages/**/dist/",
```

Comment:
It's already there. @bluwy

Reply:
And I did not reproduce that

Reply:
The CI is also passing, what env are you in?

Reply:
Ah it's the `benchmark/dist/` that's triggering it. I think it's also possible for directories like `examples/blog` to have dist when you're testing things locally. So would be nice to put a blanket ignore for `dist`.

Reply:
Got it

Reply:
`pnpm build && pnpm benchmark` will reproduce it

Reply:
fixed